challenge-03/challenge-03.js

```diff
@@ -0,0 +1,207 @@
+
+/*
+Qual a idade da pessoa? (Use a instrução para responder e comentários
+inline ao lado da instrução para mostrar qual foi a resposta retornada)
+*/
+pessoa.mostrarIdade();  // 'Olá, eu tenho 20 anos!'
+
+/*
+Qual o peso da pessoa? (Use a instrução para responder e comentários
+inline ao lado da instrução para mostrar qual foi a resposta retornada)
+*/
+pessoa.mostrarPeso(); // 'Eu peso 61Kg.'
+
+/*
+Qual a altura da pessoa? (Use a instrução para responder e comentários
+inline ao lado da instrução para mostrar qual foi a resposta retornada)
+*/
+pessoa.mostrarAltura(); // 'Minha altura é 1.55m.'
+
+/*
+Faça a `pessoa` fazer 3 aniversários.
+*/
+pessoa.fazerAniversario();
+pessoa.fazerAniversario();
+pessoa.fazerAniversario();
+
+/*
+Quantos anos a `pessoa` tem agora? (Use a instrução para responder e
+comentários inline ao lado da instrução para mostrar qual foi a resposta
+retornada)
+*/
+pessoa.idade // 23
+
+/*
+Agora, faça a `pessoa` caminhar alguns metros, invocando o método `andar` 3x,
+com metragens diferentes passadas por parâmetro.
+*/
+pessoa.andar(3);
+pessoa.andar(2);
+pessoa.andar(6);
+
+/*
+A pessoa ainda está andando? (Use a instrução para responder e comentários
+inline ao lado da instrução para mostrar qual foi a resposta retornada)
+*/
+pessoa.andando // true
+
+/*
+Se a pessoa ainda está andando, faça-a parar.
+*/
+pessoa.parar();
+
+/*
+E agora: a pessoa ainda está andando? (Use uma instrução para responder e
+comentários inline ao lado da instrução para mostrar a resposta retornada)
+*/
+pessoa.andando // false
+
+/*
+Quantos metros a pessoa andou? (Use uma instrução para responder e comentários
+inline ao lado da instrução para mostrar a resposta retornada)
+*/
+pessoa.caminhouQuantosMetros // 11
+
+/*
+Agora vamos deixar a brincadeira um pouco mais divertida! :D
+Crie um método para o objeto `pessoa` chamado `apresentacao`. Esse método deve
+retornar a string:
+- "Olá, eu sou o [NOME COMPLETO], tenho [IDADE] anos, [ALTURA], meu peso é [PESO] e, só hoje, eu já caminhei [CAMINHOU QUANTOS METROS] metros!"
+
+Só que, antes de retornar a string, você vai fazer algumas validações:
+- Se o `sexo` de `pessoa` for "Feminino", a frase acima, no início da
+apresentação, onde diz "eu sou o", deve mostrar "a" no lugar do "o";
+- Se a idade for `1`, a frase acima, na parte que fala da idade, vai mostrar a
+palavra "ano" ao invés de "anos", pois é singular;
+- Se a quantidade de metros caminhados for igual a `1`, então a palavra que
+deve conter no retorno da frase acima é "metro" no lugar de "metros".
+- Para cada validação, você irá declarar uma variável localmente (dentro do
+método), que será concatenada com a frase de retorno, mostrando a resposta
+correta, de acordo com os dados inseridos no objeto.
+*/
+pessoa.apresentacao = function(){
+	var art;
+	var idad;
+	var dist;
+	
+	if( pessoa.sexo === 'Feminino' ){
+		var art = 'a';
+	}
+	else if( pessoa.sexo === 'Masculino' ){
```

Comment:
-1
Para não precisar utilizar `else` ou `else if` (evite sempre que possível), você pode definir um valor padrão para as variáveis declaradas no início da função, e então, usar só o `if` para modificar os valores conforme necessário :)